index.html

```diff
@@ -1,229 +1,204 @@
-		synchronized Web applications. The former is a dependency of the latter, meaning kinto.js communicates with a Kinto server
-		leveraging the kinto-client.js JavaScript API. kinto-client.js supports a much broader scope of the Kinto API, while kinto.js
-		simply uses what it needs from it to synchronize collection data.</p>
-      </div>
-    </div>
-  </div>
-</section>
-<!--sub-form-->
-<section class="sub-form section-spacing text-center">
-  <div class="container">
-    <header class="section-header">
-      <h2>Want to get started?</h2>
-    </header>
-    <div class="row">
-      <div class="col-md-6 center-block col-sm-11">
-        <a href="https://kinto.readthedocs.io/en/latest/tutorials/index.html" class="btn">
-          Read the tutorial →</a>
-      </div>
-    </div>
-  </div>
-</section>
-<!--sub-form end-->
-</div>
-</div>
-
-
-<section class="benefits section-spacing text-center" id="footer">
-  <div class="container">
-    <header class="section-header">
-      <p>
-        Kinto &lt;3 React. Everything is less than three.
-        Logo by <a href="http://www.miho-stories.com/">Aymeric Faivre</a>.
-      </p>
-    </header>
-  </div>
-</section>
-
-
-<script src="js/highlight.pack.js"></script>
-<script>hljs.initHighlightingOnLoad();</script>
-<script src="js/jquery-2.js"></script>
-<script src="js/bootstrap.js"></script>
-<script src="js/waypoints.js"></script>
-<script src="js/jquery_003.js"></script>
-<script src="js/waypoints-sticky.js"></script>
-<script src="js/retina.js"></script>
-<script src="js/jquery.js"></script>
-<script src="js/jquery_002.js"></script>
-<script src="js/main.js"></script>
-
-</body>
-</html>
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>Kinto</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="assets/css/main.css" />
+    <link rel="stylesheet" href="assets/css/kinto.css" />
+    <!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
+  </head>
+  <body class="homepage">
+    <div id="page-wrapper">
+
+      <!-- Header -->
+      <div id="header-wrapper">
+        <div id="header" class="container">
+          <h1 id="logo"><img src="./images/logo.png" alt="Kinto"/></h1>
+        </div>
+
+        <!-- Hero -->
+        <section id="hero" class="container">
+          <header>
+            <h2>The backend for your next application<br>&nbsp;is already there</h2>
+          </header>
+          <p>Kinto is a generic JSON document store with<br>&nbsp;sharing and synchronisation capabilities.</p>
+          <ul class="actions">
+            <li><a href="#why" class="button">Why using Kinto?</a></li>
+          </ul>
+        </section>
+      </div>
+
+      <div class="wrapper" id="why">
+        <div class="container">
+          <article>
+            <header>
+              <h2>Don’t build silos. Own your data. Redecentralise the Web.</h2>

```

Comment:
How about "Re-decentralise"? At first glance, I thought this said "Recentralise".